src/coord_seq.rs

```diff
@@ -149,6 +149,93 @@ impl<'a> CoordSeq<'a> {
         }
     }
 
+    #[cfg(any(feature = "v3_10_0", feature = "v3_11_0"))]
```

Comment:
```suggestion
    #[cfg(feature = "v3_10_0")]
```

Any "higher" feature enables the "lower" ones recursively.